k8s/esp_echo_gke_ingress.yaml

```diff
@@ -40,7 +40,7 @@ spec:
     spec:
       containers:
       - name: esp
-        image: b.gcr.io/endpoints/endpoints-runtime:1.0
+        image: b.gcr.io/endpoints/endpoints-runtime:latest
```

Comment:
ping me once we figured out the release page to refer here. We should at least document :latest is for sample/test/dev only, not for production use